frontend/components/playlists/playlist_show.jsx

```diff
@@ -8,12 +8,16 @@ const PlaylistShow = ({
     currentPlaylist,
     params,
     history,
-    currentUser,
     playlistSongs,
+    currentUser,
+    songCardDropdownState,
     displayPlaylist,
     clearCurrent,
+    openSongCardDropdown,
+    closeSongCardDropdown,
     ... props
 }) => {
+    console.log("PLAYLIST SHOW", songCardDropdownState)
```

Comment:
nit: if this is for debugging purposes usually we remove these statements before committing